bika/lims/vocabularies/__init__.py

```diff
@@ -107,8 +107,10 @@ def __call__(self, context):
             folder = site.restrictedTraverse(folder)
             for portal_type in self.portal_types:
                 objects = list(folder.objectValues(portal_type))
-                objects = [o for o in objects if
-                           wf.getInfoFor(o, 'inactive_state') == 'active']
+                wf_ids = [x.id for x in wf.getWorkflowsFor(portal_type)]
+                if 'bika_inactive_workflow' in wf_ids:
+                    objects = [o for o in objects if
+                               wf.getInfoFor(o, 'inactive_state') == 'active']
```

Comment:
Hi Campbell, you are missing here the Client objects where the active sates are now in the primary workflow in the variable `review_state`. Please use the `api.is_active` function instead to cover them as well, see https://github.com/senaite/senaite.core/pull/901

Reply:
Please use this code:

```python
objects = list(folder.objectValues(portal_type))
objects = filter(api.is_active, objects)
```

Also see here: https://github.com/senaite/senaite.core/blob/master/bika/lims/api/__init__.py#L858